document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java

```diff
@@ -30,6 +32,9 @@
     @Mock
     private TransactionService transactionService;
 
+    private static final String RESOURCE_ID = "091174-913515-326060";
```

Comment:
Could these be made more generic? Still follow the same regex pattern, but I feel like unit tests should be as 'fake' as possible.

Reply:
Good point however this PR simply addresses the broken tests, any improvement will can be made further on

Reply:
I'll approve for now, but I'd advise this be done at some point. Even though it has no bearing on the tests, it all feels a little _too_ real.